diarize.py

```diff
@@ -0,0 +1,82 @@
+from pydub import AudioSegment
+import os
+import wget
+import json
+import torch
+from nemo.collections.asr.models.msdd_models import NeuralDiarizer
+from omegaconf import OmegaConf
+
+
+def config_setup(output_dir):
```

Comment:
Consider supplying your own, augmented version of https://raw.githubusercontent.com/NVIDIA/NeMo/main/examples/speaker_tasks/diarization/conf/inference/diar_infer_{DOMAIN_TYPE}.yaml
So you can separate any config from the code.